cmd/bpf2go/main.go

```diff
@@ -272,6 +275,15 @@ func getEnv(key, defaultVal string) string {
 	return defaultVal
 }
 
+func getBoolEnv(key string, defaultVal bool) bool {
+	val, ok := os.LookupEnv(key)
+	if !ok {
+		return defaultVal
+	}
+	result, _ := strconv.ParseBool(val)
```

Comment:
This error should be surfaced to the user, otherwise incorrect settings like `NO_STRIP=gobbledigook` will silently disable stripping.

Reply:
> This error should be surfaced to the user, otherwise incorrect settings like `NO_STRIP=gobbledigook` will silently disable stripping.

I made some of these changes, what do you think?

```suggestion
	result, err := strconv.ParseBool(val)
	if err != nil {
		fmt.Fprintf(os.Stderr, "Warning: The value '%s' passed from environment variable '%s' is undefined in strconv.ParseBool, using default value (%t).\n", val, key, defaultVal)
		return defaultVal
	}
```

What else should I change?
Do I need to use b2g.stdout instead of os.Stderr? (Need to add a parameter to pass writer)

Reply:
A warning is nice, what about just returning the error from the function instead? That means you need to add a separate block to add it to `run()` but that isn't so bad in my opinion.